src/vochain/vochain.proto

```diff
@@ -207,6 +221,58 @@ message RegisterKeyTx {
         string weight = 5;   // Weight to delegate to newKey
 }
 
+message MintTokensTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes to = 3;
+	bytes value = 4;
+}
+
+message SendTokensTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes from = 3;
+	bytes to = 4;
+	bytes value = 5;
+}
+
+message SetTransactionCostsTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	uint64 value = 3;
+}
+
+message SetAccountInfoTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	string infoURI = 3;
+}
+
+message AddDelegateForAccountTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes delegate = 3;
+}
+
+message DelDelegateForAccountTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes delegate = 3;
+}
+
+message CollectFaucetTx {
+	TxType txType = 1;
+	bytes nonce = 2;
+	FaucetPayload payload = 3;
+	bytes payloadSignature = 4;
+}
+
+message FaucetPayload {
+	bytes nonce = 1;
+	bytes to = 2;
+	uint64 value = 3;
```

Comment:
I think `value` is better than `amount` because is a more general word and is more used in the web3 space, so using value is less confusing and more standard for 3rd parties. But I just shared an opinionated thought, if after reading this you think `amount` is better I am going to change it :)

Reply:
AFAIK ethereum does not have a faucet specific payload. I think amount is a better description for this field.